### PR TITLE
Add docker compose

### PR DESCRIPTION
**Pull request by winebarrel**
Add docker compose settings so developers can test on their local machine.
Could you please review?

### CODE REVIEW

**Review by KumatomatomaKu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -700,6 +700,18 @@ raise_on_not_specified_shard_query true
...
+```sh
+docker compose up -d
+# rbenv shell 2.7.8
+bundle install
+bundle exec appraisal install
+export TEST_MYSQL_HOST=127.0.0.1
+export TEST_MYSQL_PORT=13306
+bundle exec appraisal rails6_0_6 rake spec
+```
+
```
Comment (hirocaster) [inline README.md:714]: Please add bellow meen.

`Not use appraisal if your old version ruby.  ex) Under Ruby 2.2.x`

Comment (winebarrel) [reply README.md:714]: Added annotation 🙏  5ea0751

Comment (hirocaster) [reply README.md:714]: As a Notice, it should suggest the opposite meaning.

Comment (winebarrel) [reply README.md:714]: Corrected the wording of the annotation 🙏  7a5d3c52e08dd8de0e2c4dfabaac014920ebe1f9

Comment (winebarrel) [reply README.md:714]: @hirocaster Could you please check it? 🙇

Comment (hirocaster) [reply README.md:714]: Thank you for your great work!

**Comment by winebarrel** [comment]
🙏

**Pull request merged**